### DISCUSSION TITLE
ConfigPortal doesn't load correctly, only weird text

### DISCUSSION THREAD

**Discussion by undee123** [khoih-prog/ESPAsync_WiFiManager#92] [General]
Hi,

I'm using an ESP8266 (nodemcu 1.0) with VSCode and copied the Async_ConfigOnSwitch.ino example 1:1 to find out how the library works. I'm on 1.12.2, by the way.
AP is successfully created by the Arduino and I can connect to it, but when opening the ConfigPortal I get just text, seems as not the whole configuration page can be transmitted:
`‰™fº˜@äÛ@˜@˜@Œ@àÛ@DÜ@DÜ@˜@˜@DÜ@DÜ@�@�@�@�@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@˜@˜@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@DÜ@‡Ùî|=¼�ÿ?ð§ÆKÿ ÝÅbQ@¼�ÿ?0¼�ÿ?TË!@ þÿ?0Ôœÿ?lÅ @ þÿ?0Ôœÿ?âš @0Ôœÿ?¼�ÿ?œfÿ?L›ÿ? fÿ?�fÿ?L›ÿ?Ôœÿ?� @X•ÿ?L•ÿ?Ÿÿ?�fÿ?L›ÿ?”fÿ?•fÿ?xr%@\‚ÿ?6 @ÿ?6 �bÿ?Ädÿ?"Ÿÿ?Ÿÿ?)J%@�bÿ?Üˆþ? @Šÿ?�Üÿ? ÜŸÿ?Šÿ?�bÿ?Ÿÿ?½Ö$@�Üÿ?$†ÿ?ÜŸÿ?¸Ô$@+(@$†ÿ?ÜŸÿ?¿+(@Ÿÿ?Ÿÿ?Øˆþ?£ž'@ÜŸÿ?;ä'@I@°Úÿ?°Úÿ?I@@Er UªUªœqT@wT@À @ @vÎXSEr ª@�µµõÊzaánÝÀ@ð@x@zñ@L@mÓ”írFþÿ3AFþÿÑÿ)Q9a1•ö è02 8Iq @AFþÿ@!""àó@(ã@ä !õÿ0ë00tŒBV#b!òÿ ç a!ðÿQñÿañÿ}Pf†@cP =ð=ðP3À¶³ptAp@4gæ@cPP3Àö³í Qâÿ r p@4@ãPP3ÀptAö³ð0 !Þÿ¬8HX""ÂG³hKUiK3G3ôFùÿVþVÕý . (Li …ûõJÿ?66@.ÿ? ;™”²ÿ??ì²ÿ?E;q= fde-IT,de;q=0.9,en-US;q=0.8,en;q=0.7,de-DE;q=0.6rome8… fêmaCachegin.4�./ .74, /# use,e-Controlle>O,"ache, no-store, must-revalidateeste.œ ÿ?T¡ÿ?andr0L ÿ?¼ÿ?ess-8res80f�84t ÿ?¡ÿ?age:85f4<Œ¡ÿ?€1Ú!@A8Upgrade-Insecure-Requests4 HTD<Accept-Language: coFA¼¢ÿ?flatG…D¼fKF¢ÿ?€¢ÿ? €¼NGAccept-EncodingQKgzip, deflateoid.TNANYpt-Lh.ÿ?XQl¢ÿ?€”¢ÿ?€l,ap]TX-Requested-Withq=0.bXcom.android.captiveportallogin/*;qf]´¡ÿ?€ ÿ?//€ g…bFifä¡ÿ?Ì¡ÿ?101.mgAccepteÜ£ÿ?�‡€oiô¢ÿ?|•ÿ?¤º†mMozilla/5.0 (Linux; Android 11; M2012K11AG Build/RQ3A.211001.001; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/101.0.4951.61 Mobile Safari/537.36™otext/html,appl`
Tried with 2 differenz ESP8266's.

**Comment by khoih-prog** [comment]
There is some severe issue with your installation and/or your choice of using VSCode without enough knowledge to configure correctly.

I suggest you try to use Arduino IDE first, to be sure your installation, your hardware is OK. Then moving on to VSCode after being sure everything is OK. Don't waste your and everybody time.

I've spent my time and retested the library with ESP8266 and all is OK here

---

#### Async_ConfigOnSwitch using LittleFS on ESP8266_NODEMCU_ESP12E

```
Starting Async_ConfigOnSwitch using LittleFS on ESP8266_NODEMCU_ESP12E
ESPAsync_WiFiManager v1.12.2
[WM] RFC925 Hostname = AsyncConfigOnSwitch
[WM] Set CORS Header to :  Your Access-Control-Allow-Origin
ESP Self-Stored: SSID = HueNet1, Pass = 12345678
[WM] * Add SSID =  HueNet1 , PW =  12345678
Got ESP Self-Stored Credentials. Timeout 120s for Config Portal
[WM] LoadWiFiCfgFile 
[WM] OK
[WM] stationIP = 192.168.2.186 , gatewayIP = 192.168.2.1
[WM] netMask = 255.255.255.0
[WM] dns1IP = 192.168.2.1 , dns2IP = 8.8.8.8
Got stored Credentials. Timeout 120s for Config Portal
Current Timezone is not set. Enter Config Portal to set.
[WM] * Add SSID =  HueNet1 , PW =  12345678
[WM] * Add SSID =  HueNet5 , PW =  12345678
ConnectMultiWiFi in setup
[WM] ConnectMultiWiFi with :
[WM] * Flash-stored Router_SSID =  HueNet1 , Router_Pass =  12345678
[WM] * Add SSID =  HueNet1 , PW =  12345678
[WM] * Additional SSID =  HueNet1 , PW =  12345678
[WM] * Additional SSID =  HueNet5 , PW =  12345678
[WM] Connecting MultiWifi...
[WM] WiFi connected after time:  1
[WM] SSID: HueNet1 ,RSSI= -43
[WM] Channel: 4 ,IP address: 192.168.2.111
After waiting 3.98 secs more in setup(), connection result is connected. Local IP: 192.168.2.111
[WM] freeing allocated params!

Configuration portal requested.
[WM] RFC925 Hostname = ConfigOnSwitch
[WM] Set CORS Header to :  Your Access-Control-Allow-Origin
Opening configuration portal. 
ESP Self-Stored: SSID = HueNet1, Pass = 12345678
[WM] * Add SSID =  HueNet1 , PW =  12345678
Got ESP Self-Stored Credentials. Timeout 120s for Config Portal
[WM] 
Configuring AP SSID = ESP_702FF3
[WM] AP PWD = MyESP_702FF3
[WM] AP Channel = 11
[WM] AP IP address = 192.168.4.1
[WM] HTTP server started
[WM] startConfigPortal : Enter loop
[WM] Request redirected to captive portal
[WM] Location http:// 192.168.4.1
[WM] Request redirected to captive portal
[WM] Location http:// 192.168.4.1
[WM] Request redirected to captive portal
[WM] Location http:// 192.168.4.1
```

![Selection_007](https://user-images.githubusercontent.com/57012152/168339219-3df35e6a-4a29-479a-a069-d2d05b6515ea.png)

![Selection_008](https://user-images.githubusercontent.com/57012152/168339233-82c08270-809b-4543-ae6f-f161ff83be2e.png)
